spec/models/location_spec.rb

```diff
@@ -9,10 +9,6 @@
   end
 
   describe "address" do
-    it "strips the non-digit numbers from the phone input" do
```

Comment:
Removing this example since its description appears to be copy-pasted from the preceding one, and the assertion is reproduced the following example.